.circleci/trigger_func_test.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# trigger SCK 'master' branch to introduce this new image from this commit 
+# to working version of every other component.
+ORGANIZATION=splunk
+PROJECT=splunk-connect-for-kubernetes
+BRANCH=master
+
+# Trigger functional test
+curl -X POST --header "Content-Type: application/json" \
+    -d '{"build_parameters": {"CIRCLE_JOB":"build_test", \
+                            "TRIG_BRANCH":"$CIRCLE_BRANCH", \
+                            "TRIG_PROJECT":"$CIRCLE_PROJECT_REPONAME", \
+                            "TRIG_REPO":"$CIRCLE_REPOSITORY_URL", \
+                        } \
+    }' "https://circleci.com/api/v1/project/$ORGANIZATION/$PROJECT/tree/$BRANCH?circle-token=$CIRCLE_TOKEN" >>build.json
+
+BUILD_NUM=$(jq -r .build_num build.json)
+
+# Wait until finish or maximum 20 minutes
+TIMEOUT=20
+DONE="FALSE"
+until [TIMEOUT -lt 0 || "$DONE" == "TRUE" ]; do
+    curl https://circleci.com/api/v1.1/project/github/$ORGANIZATION/$PROJECT/$BUILD_NUM?circle-token=$CIRCLE_TOKEN >build_progress.json
+    STATUS=$(jq -r .status build_progress.json)
+    if [ "$STATUS" != "running" ]; then
```

Comment:
Please handle all the other possible states - https://circleci.com/docs/2.0/workflows/#states

Reply:
Possibly write a function as the same logic will be used again below